docker-compose.yml

```diff
@@ -28,43 +28,6 @@ services:
     volumes:
       - ./dex-contracts:/app/dex-contracts
 
-  postgres:
-    image: postgres
-    ports:
-      - '5432:5432'
-    environment:
-      POSTGRES_USER: dfusion
-      POSTGRES_PASSWORD: let-me-in
-      POSTGRES_DB: dfusion
-
-  driver:
-    << : *rust-service
-    command: cargo run --bin dfusion
-    depends_on:
-      - ganache-cli
-    volumes:
-      # Don't sync target, otherwise we rebuild on every deploy
-      - ./dex-contracts:/app/dex-contracts
-      - ./dfusion_rust_core/src:/app/dfusion_rust_core/src
-      - ./driver/src:/app/driver/src
-
-  graph-listener:
-    << : *rust-service
```

Comment:
We might not need the rust-service thingy anymore either.

Reply:
It's still used by the `stablex` docker compose service. I wanted to keep this "remove only" as much as possible. I imagine once we are done removing we can refactor the CI to not use docker compose anymore at all.